dags/main_summary.py

```diff
@@ -43,6 +45,79 @@
     email=["telemetry-alerts@mozilla.com", "relud@mozilla.com", "jklukas@mozilla.com"],
     dag=dag)
 
+main_ping_bigquery_export = gke_command(
+    task_id="main_ping_bigquery_extract",
+    command=[
+        "bq",
+        "extract",
+        "--destination_format=AVRO",
+        "moz-fx-data-shared-prod.payload_bytes_decoded.telemetry_telemetry__main_v4${{ds_no_dash}}",
+        "gs://moz-fx-data-derived-datasets-parquet-tmp/export/submission_date={{ds}}/document_namespace=telemetry/document_type=main/document_version=4/*.avro"
+    ],
+    docker_image="google/cloud-sdk:263.0.0-slim",
+)
+
+main_summary_dataproc = SubDagOperator(
+    subdag=moz_dataproc_jar_runner(
+        parent_dag_name="main_summary",
+        dag_name="main_summary_dataproc",
+        default_args=default_args,
+        cluster_name="main-summary-{{ds}}",
+        image_version="1.3",
+        idle_delete_ttl="600",
+        worker_machine_type="n1-standard-8",
+        num_preemptible_workers=40,
+        optional_components=[],
+        install_component_gateway=False,
+        jar_urls=[
+            "s3a://net-mozaws-data-us-west-2-ops-ci-artifacts/mozilla/telemetry-batch-view/master/telemetry-batch-view.jar"
+        ],
+        main_class="com.mozilla.telemetry.views.MainSummaryView",
+        jar_args=[
+            "--from={{ds_nodash}}",
+            "--to={{ds_nodash}}"
+            "--bucket=gs://moz-fx-data-derived-datasets-parquet-tmp",
+            "--export-path=gs://moz-fx-data-derived-datasets-parquet-tmp/export",
+        ],
+        artifact_bucket=None,
+        gcp_conn_id="google_cloud_airflow_dataproc",
+    ),
+    task_id="main_summary_dataproc",
+    dag=dag,
+)
+
+main_summary_dataproc_bigquery_load = SubDagOperator(
+    subdag=load_to_bigquery(
+        parent_dag_name=dag.dag_id,
+        dag_name="main_summary_dataproc_bigquery_load",
+        default_args=default_args,
+        dataset_gcs_bucket="moz-fx-data-derived-datasets-parquet-tmp",
+        dataset="main_summary",
+        dataset_version="v4",
+        gke_cluster_name="bq-load-gke-1",
+        bigquery_dataset="backfill",
```

Comment:
@jasonthomas is there a better dataset I can use here until this is ready to replace the existing bigquery import?

Reply:
In what sense do mean by better? The `backfill` dataset is a temporary staging area that I've used in the past but if we prefer to use a different dataset I am not against it, we would just need to provision it.

Reply:
just checking that you think it's an acceptable dataset to use as a temporary staging area for this.